src/interpreter/ByteCode.h

```diff
@@ -241,6 +241,23 @@ class FunctionType;
     F(F64Load, double, double)       \
     F(V128Load, Vec128, Vec128)
 
+#define FOR_EACH_BYTECODE_LOAD_MULTI_OP(F) \
```

Comment:
Load multi sounds like the load reads multiple data. I think MEMIDX / MemIdx describes better that the instruction has a memory index.